src/BugsnagUnity/Native/Android/Configuration.cs

```diff
@@ -8,13 +8,14 @@ class Configuration : AbstractConfiguration
   {
     internal NativeInterface NativeInterface { get; }
 
-    internal Configuration(string apiKey) : base()
+    internal Configuration(string apiKey, bool autoNotify) : base()
```

Comment:
This implementation of `Configuration` does not override `AutoNotify` so it will also need to be set from the `autoNotify` parameter. The easiest way is probably to pass the value to `setupDefaults` and handle it there.